<ui version="4.0" >
 <class>CropDialog</class>
 <widget class="QWidget" name="CropDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Crop</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="4" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="QSpinBox" name="leftSpinBox" />
   </item>
   <item row="1" column="1" >
    <widget class="QSpinBox" name="topSpinBox" />
   </item>
   <item row="2" column="1" >
    <widget class="QSpinBox" name="widthSpinBox" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>&amp;Height:</string>
     </property>
     <property name="buddy" >
      <cstring>heightSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QSpinBox" name="heightSpinBox" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&amp;Y:</string>
     </property>
     <property name="buddy" >
      <cstring>topSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>&amp;Width:</string>
     </property>
     <property name="buddy" >
      <cstring>widthSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&amp;X:</string>
     </property>
     <property name="buddy" >
      <cstring>leftSpinBox</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leftSpinBox</tabstop>
  <tabstop>topSpinBox</tabstop>
  <tabstop>widthSpinBox</tabstop>
  <tabstop>heightSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
